src/main/java/domain/board/ChessBoard.java

```diff
@@ -3,26 +3,29 @@
 import domain.piece.Color;
 import domain.piece.Empty;
 import domain.piece.Piece;
+import domain.piece.Type;
 import domain.position.Position;
 import domain.position.Route;
+import dto.PieceDto;
 
 import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class ChessBoard {
     private final Map<Position, Piece> board;
     private Color turn;
 
-    private ChessBoard(Map<Position, Piece> board, Color color) {
-        this.board = board;
-        this.turn = color;
-    }
-
     public ChessBoard(Map<Position, Piece> board) {
         this(new HashMap<>(board), Color.WHITE);
     }
 
+    ChessBoard(Map<Position, Piece> board, Color turn) {
```

Comment:
```suggestion
    ChessBoard(final Map<Position, Piece> board, Color turn) {
```
지금 보니 final 이 누락된 부분이 곳곳에 보이네요!
망쵸 스스로의 컨벤션이라 여태 리뷰를 남기진 않고 있었는데요,
이제 슬슬 이번 미션도 마무리 되어가니 한 번 말해보자면 저는 파라미터와 지역변수에 final 처리를 해주지 않아요.
프로젝트 규모가 커지고 참여하는 개발자가 많아질수록 지키기 거의 불가능한 컨벤션이더라구요.

Reply:
이피케이 말씀에 전적으로 동의해요! 계속 억지로 지키려 한 것 같아요. 커밋을 보면 주기적으로 코드를 읽으며 final을 넣어주는 작업을 하는 데도, 놓치는 부분이 생기네요. 

이전에 크루들과 이야기해봤을 때도, 장점보다 비용이 많이드는 컨벤션이라는 확신이 서네요. 이제 final을... 안 붙여볼게요! 

앗.. 이번 코드는 유지해도 괜찮을까요?! 다 떼는 작업도 큰 비용이 들 것 같아요 😂

Reply:
당연히 남겨야죠! 장점도 확실히 있는 방식이니까요. 그리고 망쵸가 말해주신 것처럼 다 떼는 작업은 공수가 너무 많이 들어요.